mainwindow.cpp

```diff
@@ -549,6 +551,23 @@ void MainWindow::redo() {
     editor->redo();
 }
 
+// Open current map scripts in system default editor for .inc files
+void MainWindow::openInTextEditor() {
+    QProcess *process = new QProcess(this);
+    process->setWorkingDirectory(editor->project->root);
+
+    #ifdef Q_OS_DARWIN
+        QString cmd = "open ";
+    #elif defined Q_OS_LINUX
+        QString cmd = "xdg-open ";
+    #else // WINDOWS 
+        QString cmd = "cmd /c start ";

```

Comment:
I don't think this will work unless running pretmap from the command line (and not sure even then)